src/components/FormValidator.js

```diff
@@ -9,6 +9,8 @@ export default class FormValidator {
     this._errorClass = settings.errorClass;
     this._buttonElement = formEl.querySelector(this._submitButtonSelector);
     this._inputList = Array.from(formEl.querySelectorAll(this._inputSelector));
+    this._inputEls = [...this._form.querySelectorAll(this._inputSelector)];
```

Comment:
```js

    this._inputEls = [...this._form.querySelectorAll(this._inputSelector)];
    this._submitBtn = this._form.querySelector(this._submitButtonSelector);
```

Please, delete them because you've already found `this._buttonElement ` and `this._inputList`